packages/lib/src/vis/utils.ts

```diff
@@ -77,20 +76,20 @@ export function getSizeToFit(
 }
 
 export function getDomain(
-  valuesArray: NdArray<number[]> | number[],
+  valuesArray: AnyNumArray,
   scaleType: ScaleType = ScaleType.Linear,
-  errorArray?: NdArray<number[]> | number[]
+  errorArray?: AnyNumArray
```

Comment:
FYI, technically, I only had to change `useDomain` and `useDomains`, neither of which use `getDomain` and `getDomains` for memoisation reasons (`getDomain` and `getDomains` are used in the stories and in `getAxisDomain`), but I think it's important that the signatures remain in sync.